Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3791,12 +3810,13 @@ def qradar_search_retrieve_events_command(
         if end_date else False
     try:
         events, status = poll_offense_events(client, search_id, should_get_events=True, offense_id=args.get('offense_id', ''))
-    except (requests.ConnectionError, requests.Timeout, requests.RequestException) as e:
+    except (DemistoException, requests.Timeout) as e:
```

Comment:
I think that you will catch here all the exceptions DemistoException Inherit from BaseException 
I am not really sure that it is correct. You won't get any indication about errors.

Reply:
wdym?